src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -149,6 +149,23 @@ export class CellModel extends Disposable implements ICellModel {
 		return this._attachments;
 	}
 
+	addAttachment(mimeType: string, base64Encoding: string, name: string): nb.ICellAttachments {
```

Comment:
Seems odd to return the attachments from this - calls can just call the attachments getter if they want to get them after adding.

Reply:
yeah, I was sticking to the interface that Chris added, will update that.